packages/components/src/mobile/image-with-focalpoint/index.native.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
We don't import React directly. I think everything you might need from React should be available in `@wordpress/element` instead

Reply:
Thanks! Updated it to use `@wordpress/element` instead =)